packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -17,7 +17,13 @@ import type { OnboardingStep } from "@/features/onboarding/types";
 
 const EndCtaIcon = () => <Icon name="playCircle" size="small" color="grey11" />;
 
-export const OnboardingProgressStepper = () => {
+interface OnboardingProgressStepperProps {
+  hasLargeButton: boolean;
```

Comment:
💡 if we make it `buttonSize: "large" | "medium"` it's both nicer in terms of api and more flexible

Reply:
changed, also added this prop to function returning rest of the ui configuration